### PR TITLE
K8SPXC-933: Fix deprecated affinity keys

### PR DESCRIPTION
**Pull request by egegunes**
[K8SPXC-933](https://jira.percona.com/browse/K8SPXC-933) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

See:
https://kubernetes.io/docs/reference/labels-annotations-taints/#failure-domainbetakubernetesioregion
https://kubernetes.io/docs/reference/labels-annotations-taints/#failure-domainbetakubernetesiozone

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scheduled-backup | failed |
| recreate | passed |
| upgrade-haproxy | passed |
| scaling | passed |
| big-data | passed |
| storage | passed |
| haproxy | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| self-healing-chaos | failed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| limits | passed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | failed |
| demand-backup | passed |
| smart-update | failed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | failed |
| affinity | failed |
| one-pod | passed |
| pitr | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |
| upgrade-consistency | failed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1079/commits/2aad9a465f63a4953292c27ef98777976b82c531
image: `perconalab/percona-xtradb-cluster-operator:PR-1079-2aad9a46`

**Comment by delgod** [comment]
link to ticket, please

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| storage | passed |
| recreate | passed |
| big-data | passed |
| scheduled-backup | passed |
| upgrade-haproxy | passed |
| haproxy | passed |
| self-healing | passed |
| self-healing-chaos | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| self-healing-advanced | passed |
| self-healing-advanced-chaos | passed |
| limits | passed |
| demand-backup | passed |
| smart-update | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| operator-self-healing-chaos | passed |
| affinity | failed |
| one-pod | passed |
| pitr | passed |
| auto-tuning | passed |
| upgrade-consistency | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1079/commits/5387eaf01d0342475a7de5f7b6e924ae2807980a
image: `perconalab/percona-xtradb-cluster-operator:PR-1079-5387eaf0`

**Review by hors — CHANGES_REQUESTED** [review]
please check affinity test

File: pkg/apis/pxc/v1/pxc_types.go [hunk 1]
```diff
@@ -956,10 +956,10 @@ func (cr *PerconaXtraDBCluster) ConfigHasKey(section, key string) (bool, error)
 const AffinityTopologyKeyOff = "none"
 
 var affinityValidTopologyKeys = map[string]struct{}{
-	AffinityTopologyKeyOff:                     {},
-	"kubernetes.io/hostname":                   {},
-	"failure-domain.beta.kubernetes.io/zone":   {},
-	"failure-domain.beta.kubernetes.io/region": {},
+	AffinityTopologyKeyOff:          {},
+	"kubernetes.io/hostname":        {},
+	"topology.kubernetes.io/zone":   {},
```
Comment (hors) [inline pkg/apis/pxc/v1/pxc_types.go:961]: Should we change it for 1.11.0 only?

Comment (delgod) [reply pkg/apis/pxc/v1/pxc_types.go:961]: yes

**Review by delgod — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| upgrade-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/upgrade-haproxy.log) |
| scheduled-backup | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/scheduled-backup.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/storage.log) |
| big-data | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/big-data.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/scaling.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/recreate.log) |
| cross-site | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/cross-site.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/haproxy.log) |
| scaling-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/scaling-proxysql.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/self-healing.log) |
| self-healing-chaos | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/self-healing-chaos.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/init-deploy.log) |
| restore-to-encrypted-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/restore-to-encrypted-cluster.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/security-context.log) |
| upgrade-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/upgrade-proxysql.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/limits.log) |
| self-healing-advanced | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/self-healing-advanced.log) |
| self-healing-advanced-chaos | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/self-healing-advanced-chaos.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/demand-backup.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/monitoring-2-0.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/operator-self-healing.log) |
| operator-self-healing-chaos | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/operator-self-healing-chaos.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/smart-update.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/affinity.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/one-pod.log) |
| auto-tuning | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/auto-tuning.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/pitr.log) |
| proxysql-sidecar-res-limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/proxysql-sidecar-res-limits.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/users.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/upgrade-consistency.log) |
| demand-backup-encrypted-with-tls | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/demand-backup-encrypted-with-tls.log) |
| tls-issue-self | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/tls-issue-self.log) |
| tls-issue-cert-manager | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/tls-issue-cert-manager.log) |
| tls-issue-cert-manager-ref | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/tls-issue-cert-manager-ref.log) |
| We run 34 out of 36|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1079/commits/71fdb7cdcac078da66945b24f4fb393dbb66da12
image: `perconalab/percona-xtradb-cluster-operator:PR-1079-71fdb7cd`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/storage.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/scheduled-backup.log) |
| big-data | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/big-data.log) |
| cross-site | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/cross-site.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/haproxy.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/self-healing.log) |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/scaling.log) |
| upgrade-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/upgrade-haproxy.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/init-deploy.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/recreate.log) |
| self-healing-chaos | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/self-healing-chaos.log) |
| scaling-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/scaling-proxysql.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/limits.log) |
| restore-to-encrypted-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/restore-to-encrypted-cluster.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/security-context.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/demand-backup.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/monitoring-2-0.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/pitr.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/affinity.log) |
| demand-backup-encrypted-with-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/demand-backup-encrypted-with-tls.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/one-pod.log) |
| auto-tuning | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/auto-tuning.log) |
| proxysql-sidecar-res-limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/proxysql-sidecar-res-limits.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/users.log) |
| tls-issue-self | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/tls-issue-self.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/tls-issue-cert-manager.log) |
| upgrade-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/upgrade-proxysql.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/smart-update.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/upgrade-consistency.log) |
| self-healing-advanced | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/self-healing-advanced.log) |
| self-healing-advanced-chaos | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/self-healing-advanced-chaos.log) |
| tls-issue-cert-manager-ref | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/tls-issue-cert-manager-ref.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/operator-self-healing.log) |
| operator-self-healing-chaos | [failed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/operator-self-healing-chaos.log) |
| validation-hook | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/validation-hook.log) |
| proxy-protocol | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/proxy-protocol.log) |
| We run 36 out of 36|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1079/commits/71fdb7cdcac078da66945b24f4fb393dbb66da12
image: `perconalab/percona-xtradb-cluster-operator:PR-1079-71fdb7cd`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/scaling.log) |
| cross-site | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/cross-site.log) |
| scheduled-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/scheduled-backup.log) |
| scaling-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/scaling-proxysql.log) |
| haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/haproxy.log) |
| recreate | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/recreate.log) |
| storage | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/storage.log) |
| big-data | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/big-data.log) |
| security-context | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/security-context.log) |
| init-deploy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/init-deploy.log) |
| restore-to-encrypted-cluster | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/restore-to-encrypted-cluster.log) |
| self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/self-healing.log) |
| upgrade-haproxy | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/upgrade-haproxy.log) |
| limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/limits.log) |
| demand-backup | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/demand-backup.log) |
| self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/self-healing-chaos.log) |
| pitr | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/pitr.log) |
| monitoring-2-0 | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/monitoring-2-0.log) |
| demand-backup-encrypted-with-tls | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/demand-backup-encrypted-with-tls.log) |
| affinity | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/affinity.log) |
| one-pod | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/one-pod.log) |
| auto-tuning | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/auto-tuning.log) |
| proxysql-sidecar-res-limits | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/proxysql-sidecar-res-limits.log) |
| users | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/users.log) |
| tls-issue-self | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/tls-issue-self.log) |
| tls-issue-cert-manager | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/tls-issue-cert-manager.log) |
| tls-issue-cert-manager-ref | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/tls-issue-cert-manager-ref.log) |
| validation-hook | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/validation-hook.log) |
| proxy-protocol | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/proxy-protocol.log) |
| upgrade-proxysql | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/upgrade-proxysql.log) |
| smart-update | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/smart-update.log) |
| upgrade-consistency | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/upgrade-consistency.log) |
| self-healing-advanced | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/self-healing-advanced.log) |
| self-healing-advanced-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/self-healing-advanced-chaos.log) |
| operator-self-healing | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/operator-self-healing.log) |
| operator-self-healing-chaos | [passed](https://percona-jenkins-artifactory-public.s3.amazonaws.com/cloud-pxc-operator/PR-1079/71fdb7cd/operator-self-healing-chaos.log) |
| We run 36 out of 36|

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/1079/commits/71fdb7cdcac078da66945b24f4fb393dbb66da12
image: `perconalab/percona-xtradb-cluster-operator:PR-1079-71fdb7cd`

**Review by hors — APPROVED** [review]

**Pull request merged**